source/elements/dpcpp/source/index.rst

```diff
@@ -49,151 +48,99 @@ some additional extensions being required, and some DPC++ extensions no longer
-   Required work-group size     `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   Standard layout relaxed      `SYCL 2020 provisional`_  Required              Required              Required              NA [#na]_
-   Unified Shared Memory        `SYCL 2020 provisional`_  Required [#USM]_      Required [#USM]_      Required [#USM]_      `usm <https://github.com/intel/llvm/tree/sycl/sycl/test/usm>`__
    `Accessor properties`_       DPC++ extension           Required [#aprop]_    Required [#aprop]_    Required [#aprop]_    NA [#na]_
+   `Device-side assert`_        DPC++ extension           Required              Required              Required              NA [#na]_
    `CXX standard library`_      DPC++ extension           Required              Required              Not required [#tmp]_  NA [#na]_
    `Data flow pipes`_           DPC++ extension           Not required          Not required          Required              `fpga_tests <https://github.com/intel/llvm/tree/sycl/sycl/test/fpga_tests>`__
    `Enqueued barriers`_         DPC++ extension           Required              Required              Required              NA [#na]_
-   `Group algorithms`_          DPC++ extension           Required              Required              Not required [#tmp]_  NA [#na]_
-   `Group mask`_                DPC++ extension           Not required [#tmp]_  Not required [#tmp]_  Not required [#tmp]_  NA [#na]_
-   `Parallel for shortcuts`_    DPC++ extension           Required              Required              Required              NA [#na]_
-   `Pinned memory property`_    DPC++ extension           Required              Required              Required              NA [#na]_
-   `Reductions`_                DPC++ extension           Required [#redc]_     Required [#redc]_     Not required [#tmp]_  NA [#na]_
+   `Extended atomics`_          DPC++ extension           Required              Required              Required              NA [#na]_
+   `Filter selector`_           DPC++ extension           Required              Required              Required              NA [#na]_
+   `FPGA LSU controls`_         DPC++ extension           Not required          Not required          Required              NA [#na]_
+   `FPGA memory channel`_       DPC++ extension           Not required          Not required          Required              NA [#na]_
+   `FPGA register`_             DPC++ extension           Not required          Not required          Required              NA [#na]_
+   `FPGA selector`_             DPC++ extension           Required              Required              Required              NA [#na]_
+   `GPU device info`_           DPC++ extension           Required              Required              Required              NA [#na]_
+   `Level zero backend`_        DPC++ extension           Not required          Not required          Not required          NA [#na]_
```

Comment:
This makes no sense to have an entry in this table with "Not required" for all columns.  If it's truly not required for any device, should we remove it?

What is the logic for saying it is not required for GPU?  Is it because another vendor is allowed to implement GPU with a different backend?

I suggest either removing this entry or adding a note that says something like "required if the device backend is Level Zero".

Reply:
Yes, to enable other vendor's backends.  If we require L0 for FPGA or GPU, I think we require all instances to sit on top of and interop with L0.  I'll add the note you suggest